packages/dotcom-ui-base-styles/styles.scss

```diff
@@ -1,3 +1,5 @@
+$system-code: 'page-kit-base-styles' !default;
+
 @import 'n-ui-foundations/main';
```

Comment:
I reckon let's use `@use` here 😄

Reply:
And a big TODO comment about eventually sorting out `n-ui-foundations`

Reply:
Fixed in [this commit](https://github.com/Financial-Times/dotcom-page-kit/pull/769/commits/2a306f27e202aa446adda2f941fb796243f59fe7) and [this commit](https://github.com/Financial-Times/dotcom-page-kit/pull/769/commits/e24e547b096aec8aec6e5aa3bd9235cdc93dd1ec) respectively.